pyproject.toml

```diff
@@ -18,7 +18,7 @@ ensembl_genes = "ensembl_genes.ensembl_genes:Commands.command"
 # https://python-poetry.org/docs/versions/
 [tool.poetry.dependencies]
 python = "^3.9"
-fastparquet = "^0.7.1"
+pyarrow = "^7.0.0"
```

Comment:
Noting that this change increased the install dependencies step runtime on CI by ~30 seconds to 1:42 ([before](https://github.com/related-sciences/ensembl-genes/runs/5016534654?check_suite_focus=true) & [after](https://github.com/related-sciences/ensembl-genes/runs/5165338001?check_suite_focus=true)).